### PR TITLE
Formulate response-body for vlan-interface

### LINKED ISSUE #90: To formulate the parameters of configured-lan-port-role-list
- [ ] Filter all the wire-interfaces from the RequestForProvidingAcceptanceDataCausesReadingLtpStructure
For each wire-interface find out the client ethernet-container, 

- [ ] **Formulate the interface-name**
Perform the callback RequestForProvidingAcceptanceDataCausesDeterminingTheLanPortRole.OriginalLtpName
Config file : alap-1-0-0-op-c-is-mwdi-1-0-0-201 + alap-1-0-0-string-p-007
url : /core-model-1-4:network-control-domain=cache/control-construct={mount-name}/logical-termination-point={uuid}/ltp-augment-1-0:ltp-augment-pac?fields=original-ltp-name

- [ ] **Formulate the vlan-interface-kind**
For the figured ethernet-container , find the mapped client vlan-interfaces
Perform the callback RequestForProvidingAcceptanceDataCausesDeterminingTheLanPortRole. VlanInterfaceKind
alap-1-0-0-op-c-is-mwdi-1-0-0-261 + alap-1-0-0-string-p-010
/core-model-1-4:network-control-domain=cache/control-construct={mount-name}/logical-termination-point={uuid}/layer-protocol={local-id}/vlan-interface-1-0:vlan-interface-pac/vlan-interface-configuration?fields=interface-kind

- [ ] **Formulate the serving-ethernet-container-status**
For the figured ethernet-container, retrieve the status information  
perform the callback RequestForProvidingAcceptanceDataCausesDeterminingTheLanPortRole.EthernetContainerStatus
alap-1-0-0-op-c-is-mwdi-1-0-0-222 + alap-1-0-0-string-p-009
/core-model-1-4:network-control-domain=cache/control-construct={mount-name}/logical-termination-point={uuid}/layer-protocol={local-id}/ethernet-container-2-0:ethernet-container-pac/ethernet-container-status?fields=interface-status

### LINKED ISSUE #91: To formulate the parameters of configured-wan-port-role-list
- [ ] Filter all the air-interfaces from the RequestForProvidingAcceptanceDataCausesReadingLtpStructure

For each air-interface find out the client ethernet-container
- [ ] Formulate the interface-name
Perform the callback RequestForProvidingAcceptanceDataCausesDeterminingTheWanPortRole.OriginalLtpName
Config file : alap-1-0-0-op-c-is-mwdi-1-0-0-201 + alap-1-0-0-string-p-011
url : /core-model-1-4:network-control-domain=cache/control-construct={mount-name}/logical-termination-point={uuid}/ltp-augment-1-0:ltp-augment-pac?fields=original-ltp-name

- [ ] Formulate the vlan-interface-kind
For the figured ethernet-container , find the mapped client vlan-interfaces
Perform the callback RequestForProvidingAcceptanceDataCausesDeterminingTheWanPortRole. VlanInterfaceKind
alap-1-0-0-op-c-is-mwdi-1-0-0-261 + alap-1-0-0-string-p-012
/core-model-1-4:network-control-domain=cache/control-construct={mount-name}/logical-termination-point={uuid}/layer-protocol={local-id}/vlan-interface-1-0:vlan-interface-pac/vlan-interface-configuration?fields=interface-kind

- [ ] Formulate the serving-ethernet-container-status
For the figured ethernet-container, retrieve the status information
Perform the callback RequestForProvidingAcceptanceDataCausesDeterminingTheWanPortRole.EthernetContainerStatus
alap-1-0-0-op-c-is-mwdi-1-0-0-222 + alap-1-0-0-string-p-008
/core-model-1-4:network-control-domain=cache/control-construct={mount-name}/logical-termination-point={uuid}/layer-protocol={local-id}/ethernet-container-2-0:ethernet-container-pac/ethernet-container-status?fields=interface-status

### PR DESCRIPTION
**Pull request by IswaryaaS**
Fixes #90 
Fixes #91

### CODE REVIEW

File: server/service/individualServices/LtpStructureUtility.js [hunk 1]
```diff
@@ -9,12 +9,70 @@ const onfAttributes = require('onf-core-model-ap/applicationPattern/onfModel/con
...
+
+/**
+ * This function fetches LTP instance from LtpStructure if uuid is equal to given.
+ * @param {String} uuid identifier of the logical-termination-point instance.
+ * @param {Object} ltpStructure control construct.
+ * @return {Object} ltp that matches the given uuid.
+ */
+async function getLtpForUuidFromLtpStructure(uuid, ltpStructure) {
+    let ltp = {};
+    let ltpList = ltpStructure["core-model-1-4:control-construct"][onfAttributes.CONTROL_CONSTRUCT.LOGICAL_TERMINATION_POINT];
```
Comment (vanithavalluripalli9) [inline server/service/individualServices/LtpStructureUtility.js:30]: As controlConstruct is an array can we change it to ["core-model-1-4:control-cosntruct"][0]

**Review by vanithavalluripalli9 — CHANGES_REQUESTED** [review]

File: server/service/individualServices/LtpStructureUtility.js (hunk 1 above)
Comment (IswaryaaS) [reply server/service/individualServices/LtpStructureUtility.js]: Thanks for pointing it out @vanithavalluripalli9 . Updated CC to array.

File: server/service/individualServices/ReadVlanInterfaceData.js [hunk 2]
```diff
@@ -1,28 +1,340 @@
...
+ * Fetches original-ltp-name for ethernet-container ltp associated to chosen wire/air layer
+ * @param {String} mountName Identifier of the device at the Controller
+ * @param {Object} ltpStructure ControlConstruct provided from cache
+ * @param {Object} requestHeaders Holds information of the requestHeaders like Xcorrelator , CustomerJourney,User etc.
+ * @param {Integer} traceIndicatorIncrementer traceIndicatorIncrementer to increment the trace indicator
+ * @param {Object} ltp chosen ltp instance of wire/air layer
+ * @param {Object} clientAndFieldParams Holds information of the query and field parameters to use in request url
+ * @returns {Object} returns original-ltp-name value and traceIndicatorIncrementer
+ */
+async function fetchOriginalLtpName(mountName, ltpStructure, requestHeaders, traceIndicatorIncrementer, ltp, clientAndFieldParams) {
```
Comment (PrathibaJee) [inline server/service/individualServices/ReadVlanInterfaceData.js:273]: function name shall be "fetchOriginalLtpNameOfEthernetContainer".

File: server/service/individualServices/ReadVlanInterfaceData.js [hunk 3]
```diff
@@ -1,28 +1,340 @@
...
+  let originalLtpNameResponse = {};
+  let pathParamList = [];
+  let interfaceListForOriginalLtpName = [PURE_ETHERNET_STRUCTURE.MODULE + PURE_ETHERNET_STRUCTURE.LAYER_PROTOCOL_NAME, ETHERNET_CONTAINER.MODULE + ETHERNET_CONTAINER.LAYER_PROTOCOL_NAME];
```
Comment (PrathibaJee) [inline server/service/individualServices/ReadVlanInterfaceData.js:276]: For AirInterface, the client shall be PureEthernetStructure or HybridMicrowaveStructure. Kindly please update the code to support in both the cases. Thanks.

**Review by PrathibaJee — CHANGES_REQUESTED** [review]
Overall it looks good. Well done.
I have included few minor review comments , kindly please have a look. 
Many thanks.

**Review by PrathibaJee — APPROVED** [review]
Many thanks for fixing the review comments. approving the changes.

**Review by vanithavalluripalli9 — APPROVED** [review]

**Pull request merged**